src/Runtime/platform.h

```diff
@@ -89,6 +89,10 @@ typedef jmp_t jtbl_t[];
 /// A @c void callback with no arguments.
 typedef void (*Event)(void);
 
+#if defined(__PPCGEKKO__)
+#define GEKKO
+#endif
+
```

Comment:
Shouldn't we just find/replace instead?

Reply:
Well it'll need to be defined manually by gcc et al so I think it makes sense to abstract it behind GEKKO.

Reply:
Yeah but then they'd need to manually define `GEKKO`, right? So they can just manually define `__PCCGEKKO__` instead.

Reply:
Ok fair enough, I'll just replace them all.